package.json

```diff
@@ -8,13 +8,14 @@
     "prepare": "husky install tools/husky",
     "build": "npm run build --workspaces --if-present",
     "flow": "flow",
+    "typescript": "npm run typescript --workspaces --if-present",
```

Comment:
This is really a test, and not a monorepo task. You should be able to revert the changes to this file and change the script name in the new package